nixos/doc/manual/release-notes/rl-2311.section.md

```diff
@@ -91,6 +91,8 @@
 
 - [ZITADEL](https://zitadel.com), a turnkey identity and access management platform. Available as [services.zitadel](#opt-services.zitadel.enable).
 
+- [OneDriver](https://github.com/jstaf/onedriver), a network filesystem that gives your computer direct access to your files on Microsoft OneDrive.
+
```

Comment:
I'm not sure a release-notes entry is necessary if you're only adding a package.

But it might be useful to add a module - there is one for [onedrive](https://github.com/MrcJkb/nixpkgs/blob/4362b0fb22b54b1cb15c2d34a39f9ff5b5d708dd/nixos/modules/services/networking/onedrive.nix#L43).

Reply:
I removed the release note entry. I'm not sure having a module would be a good idea since the configuration has to be done from the UI (for the OAuth2 token authorization) and the services can be configured easily after the first configuration..

Reply:
abraunegg/onedrive needs to authorize too, so I think a module would be fine. But it's definitely not a hard requirement 😃

Reply:
Ok, if you don't mind I'll just stick with a simple derivation then for now :smile: